### ISSUE TITLE
Flows not showing up

### ISSUE THREAD

**Issue by xxcite81** [goatatwork/ntopng-nprobe-docker#1]
Hi,
I have ran the docker containers with docker compose and get the  web interface. I have mikrotik pointing to the server:2055udp but no flows show up.
I have checked a  few things such as that the port 2055udp is open but still cant get anything.

Can you assist

**Comment by cltnoc** [comment]
@xxcite81 I am facing the same issue, whether your issue was solved?? if yes please share the solution.
I can see UDP packets reaching port 2055 on packet capturing however the ntop-ng interface show no packets received.

**Comment by xxcite81** [comment]
Yes I did get it resolved. I was using Mikrotik and did the TrafficFlow. 
In my targets I had a source of 0.0.0.0 and destination of my ntop container. I saw the packets hitting the poet but they would not reach the ntop. I switched the source to the IP of the mikrotik and it worked., Note it will only collect like 10 minutes and stop because I didn't until a license.

I have since switched to a trial of SolarWinds which gives 30 days collection.
